src/style.css

```diff
@@ -1 +1,31 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). It's better to style elements using classes to maintain consistency and to avoid potential specificity issues.